### PR TITLE
Use cache for lint scripts

### PR DESCRIPTION
**Pull request by rdwz**
#### Description of Change

> $ eslint --help
> ...
> Caching:
>   --cache                          Only check changed files - default: false
> ...

> $ prettier --help
> ...
>  --cache                  Only format changed files. Cannot use with --stdin-filepath.
>                           Defaults to false.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-node_modules
-archives/
-yarn-error.log
+.eslintcache
+.idea
 .pnpm-debug.log
-server*.log
-*.png
-*.jpg
-*.jpeg
```
Comment (SpecialAro) [inline .gitignore:10]: any reason for some of this deletion ?

**Review by SpecialAro — CHANGES_REQUESTED** [review]

**Review by vraravam — COMMENTED** [review]
I dont agree with the caching. We (as maintainers) have seen innumerable times where a merged code was also not linted/formatted properly. So, only processing the changed files will not take that into account. Also, the time taken for processing all files (as opposed to only the changed files) is quite bearable for now.

File: .gitignore (hunk 1 above)
Comment (vraravam) [reply .gitignore:10]: i think the deleted lines are just moved.

**Comment by vraravam** [comment]
Closing

**Pull request closed without merging**